### PR TITLE
added JavaApi for loadCoverage

### PR DESCRIPTION
**Pull request by akmorrow13**
Resolves https://github.com/bigdatagenomics/adam/issues/1420

### CODE REVIEW

**Comment by fnothaft** [comment]
Jenkins, retest this please.

**Comment by akmorrow13** [comment]
Adding tests.

**Comment by akmorrow13** [comment]
Jenkins, test this please.

File: adam-apis/src/main/scala/org/bdgenomics/adam/apis/java/JavaADAMContext.scala [hunk 1]
```diff
@@ -84,6 +84,16 @@ class JavaADAMContext(val ac: ADAMContext) extends Serializable {
   }
 
   /**
+   * Loads in a coverage file. This method can load BED, NARROWPEAK, and ADAM files.
```
Comment (heuermh) [inline adam-apis/src/main/scala/org/bdgenomics/adam/apis/java/JavaADAMContext.scala]: NARROWPEAK &rarr; NarrowPeak as used elsewhere

What about the other feature formats (i.e. GFF3, GTF/GFF2, and IntervalList) supported by `loadFeatures`?

**Comment by heuermh** [comment]
Thank you, @akmorrow13!

**Pull request merged**